components/src/dynamo/common/backend/sample_engine.py

```
@@ -93,32 +107,81 @@ async def from_args(
+        while not self._publish_stop.is_set():
+            try:
+                kind, payload = self._publish_queue.get(timeout=0.1)
+            except queue.Empty:
+                continue
+            try:
+                if kind == "stored":
+                    publisher.publish_stored(**payload)
+                elif kind == "removed":
+                    publisher.publish_removed(**payload)
+            except Exception:
+                continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)